src/form/bf_mx.jl

```diff
@@ -9,6 +9,15 @@ end
 ""
 function variable_mc_voltage(pm::AbstractUBFModels; kwargs...)
     variable_mc_voltage_prod_hermitian(pm; kwargs...)
+    #enforce PSDness of leaf nodes that are not captured otherwise
+    for nw in _PMs.nw_ids(pm)
+        allbuses = Set(_PMs.ids(pm, nw, :bus))
+        startingbuses = Set(i for (l,i,j)  in _PMs.ref(pm, nw, :arcs_from))
+        leafnodes = setdiff(allbuses, startingbuses)
+        for i in leafnodes
+            constraint_mc_voltage_psd(pm, nw, i)
+        end
+    end
```

Comment:
I think the loop over `nw` should happen in the problem definition, should it not?